plugins/auth-backend/config.d.ts

```diff
@@ -185,6 +185,11 @@ export interface Config {
       /** @visibility frontend */
       cfaccess?: {
         teamName: string;
+        /** @visibility secret */
+        serviceTokens?: Array<{
```

Comment:
Is it possible to reference the `ServiceToken` type I create in plugins/auth-backend/src/providers/cloudflare-access/provider.ts if I `export` it?

Reply:
It's probably better to keep this explicit (i.e. to not import that)